modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view_suggestions.jsp

```diff
@@ -90,6 +89,10 @@ request.setAttribute("view_suggestions.jsp-searchContainer", kbCommentsSearchCon
 
 		orderColumns.put("modified-date", "modified-date");
 		orderColumns.put("user-name", "user-name");
+
+		PortletURL sortURL = PortletURLUtil.clone(currentURLObj, liferayPortletResponse);
+
+		sortURL.setParameter("storeOrderByPreference", Boolean.TRUE.toString());
```

Comment:
Ok, entendido. Lo dejamos así entonces. 

Sergio

> El 2 jun 2016, a las 0:37, Roberto Díaz <notifications@github.com> escribió:
> 
> In modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view_suggestions.jsp:
> 
> > @@ -90,6 +89,10 @@ request.setAttribute("view_suggestions.jsp-searchContainer", kbCommentsSearchCon
> >  
> >  		orderColumns.put("modified-date", "modified-date");
> >  		orderColumns.put("user-name", "user-name");
> > +
> > +		PortletURL sortURL = PortletURLUtil.clone(currentURLObj, liferayPortletResponse);
> > +
> > +		sortURL.setParameter("storeOrderByPreference", Boolean.TRUE.toString());
> si usamos la currentURL "a pelo" vamos propagar el parametro en caso de que llegue a false, de tal manera que los enlaces de sort tambien lo llevaran como false.
> 
> En el caso del sort, siempre vamos a querer que este a true, por eso lo seteo asi siempre
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>